[BUG] Add release notes summary for new Timeline templates (#1971)
elastic/security-docs: docs/release-notes/8.2.asciidoc

* Adds the *Blocklist enabled* toggle to Malware protection settings ({pull}127031[#127031]).
* Updates MITRE ATT&CK mappings for detection rules to v10.1 ({pull}126288[#126288]).
* Adds an Advanced Settings toggle to turn off `read` privilege warnings for detection rules using a remote cross-cluster search (CCS) index pattern ({pull}124459[#124459]).

> * Adds new Timeline templates ({pull}125172[#125172]).

Author: ```suggestion
* Added four more Timeline templates that are focused on key event categories to provide relevant alert data and assist with investigation and resolution efforts ({pull}125172[#125172]).
```

> * Adds four new Timeline templates that are focused on key event categories to provide relevant alert data and assist with investigation and resolution efforts ({pull}125172[#125172]).

Reviewer: ```suggestion
* Adds four new Timeline Templates that are focused on key event categories to provide relevant alert data and assist with investigation and resolution efforts ({pull}125172[#125172]).
```

> * Adds four new Timeline Templates that are focused on key event categories to provide relevant alert data and assist with investigation and resolution efforts ({pull}125172[#125172]).

Author: ```suggestion
* Adds four new Timeline templates that are focused on key event categories to provide relevant alert data and assist with investigation and resolution efforts ({pull}125172[#125172]).
```

> * Added four more Timeline templates that are focused on key event categories to provide relevant alert data and assist with investigation and resolution efforts ({pull}125172[#125172]).

Reviewer 2: The rest of our RNs are written in the present tense. 
```suggestion
* Adds four new Timeline templates that are focused on key event categories to provide relevant alert data and assist with investigation and resolution efforts ({pull}125172[#125172]).
```

* Adds the *Blocklist enabled* toggle to Malware protection settings ({pull}127031[#127031]).
* Updates MITRE ATT&CK mappings for detection rules to v10.1 ({pull}126288[#126288]).
* Adds an Advanced Settings toggle to turn off `read` privilege warnings for detection rules using a remote cross-cluster search (CCS) index pattern ({pull}124459[#124459]).

> * Adds four new Timeline Templates that are focused on key event categories to provide relevant alert data and assist with investigation and resolution efforts ({pull}125172[#125172]).

Reviewer 2: @DefSecSentinel `Timeline templates` are written as sentence case in our docs. Examples [here](https://www.elastic.co/guide/en/security/current/timeline-templates-ui.html).
